Domain/DoctrineAclCache.php

```diff
@@ -25,7 +25,7 @@
  */
 class DoctrineAclCache implements AclCacheInterface
 {
-    const PREFIX = 'sf2_acl_';
+    const PREFIX = 'sf3_acl_';
```

Comment:
@fabpot commits ready, I can close these PRs and open the new ones with `sf_` prefix when you agree with this change